### ISSUE TITLE
[Chore] Revisit menu items

### ISSUE THREAD

**Issue by kyungeunni** [elastic/synthetics-recorder#363]
### Task summary

We have several features in the menu, but most of them are not usable/applicable(e.g. none of the menu items under Edit works). Also, users might have [falsy expectations](https://github.com/elastic/synthetics-recorder/issues/14#issuecomment-1410188463) on the functionality when the menu items are not working or think of it as a bug. We should audit the items and remove the ones are not relevant.

**Comment by kyungeunni** [comment]
Turns out simple copy/paste hotkeys won't work when a text field is focused if we don't add them to the menu in Electron App, and that's why we've [added](https://github.com/elastic/synthetics-recorder/pull/50) Edit menu with common hotkeys. We can make it better by doing some customization but I wonder if it's worth the effort. (I think the same reason goes to [this feature ](https://github.com/elastic/synthetics-recorder/issues/14#issuecomment-1410188463) we've added for macOS.

I'd suggest we change the focus of this issue to revisit menu items that make sense per OS. For example, the first menu (labeled as the app name) only works on Mac but not on others.
